tests/python/gpu/test_forward.py

```diff
@@ -24,11 +24,13 @@
 sys.path.insert(0, os.path.join(curr_path, '../unittest'))
 from common import setup_module, with_seed, teardown
 from mxnet.gluon import utils
+import tarfile
 
 def _get_model():
     if not os.path.exists('model/Inception-7-symbol.json'):
-        download('http://data.mxnet.io/models/imagenet/inception-v3.tar.gz', dirname='model')
```

Comment:
Are you changing the behaviour here? It seems like we are not writing to model/ but into the root.

Reply:
it works as expected, tar is not available on windows. The test would fail otherwise.

Reply:
I mean that we are now extracting to a different path (the root of the mxnet source) opposed to the subdirectory "models".